packages/metrics/tests/e2e/basicFeatures.decorator.test.functionCode.ts

```diff
@@ -1,6 +1,6 @@
 import { Metrics, MetricUnits } from '../../src';
 import { Context } from 'aws-lambda';
-import { LambdaInterface } from '../../examples/utils/lambda/LambdaInterface';
+import { LambdaInterface } from '@aws-lambda-powertools/commons';
```

Comment:
Won't this use the published version instead of local version?

Reply:
As far as I understand, given that we are inside the NPM workspace & it uses the only lock file that is present at the root, then no, it should get the local one (see screenshot below):

Reply:
![image](https://user-images.githubusercontent.com/7353869/184903080-e451398c-3c5e-4102-b2da-1da35bbd5a0b.png)